curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/access-multi-dimensional-arrays-with-indexes.md

```diff
@@ -21,13 +21,11 @@ const arr = [
   [[10, 11, 12], 13, 14]
 ];
 
-arr[3];
-arr[3][0];
-arr[3][0][1];
+arr[3]; // [[10, 11, 12], 13, 14]
+arr[3][0]; // [10, 11, 12]
+arr[3][0][1]; // 11
```

Comment:
I really hate these dummy examples that do nothing when run. Perhaps something like this?

```suggestion
const subarray_3   = arr[3]; // [[10, 11, 12], 13, 14]
const subarray_3_0 = arr[3][0]; // [10, 11, 12]
const entry_3_0_1  = arr[3][0][1]; // 11
```